keel-web/keel-web.gradle.kts

```diff
@@ -28,8 +29,7 @@ dependencies {
   implementation("com.netflix.spinnaker.fiat:fiat-api:${property("fiatVersion")}")
   implementation("com.netflix.spinnaker.fiat:fiat-core:${property("fiatVersion")}")
   implementation("net.logstash.logback:logstash-logback-encoder")
-  implementation("org.springdoc:springdoc-openapi-webmvc-core")
-  implementation("org.springdoc:springdoc-openapi-kotlin")
+  implementation("io.swagger.core.v3:swagger-annotations:2.1.2")
```

Comment:
No more Springdoc OpenAPI but since we use some Swagger annotations on our controller endpoints I added that dependency explicitly (we no longer get it transitively)